src/bootstrap/replication/service.rs

```diff
@@ -271,30 +289,42 @@ impl ReplicatorBootstrap {
         // Create tasks for each shard to run in parallel
         let mut shard_tasks = JoinSet::new();
 
-        for shard_id in shard_ids {
-            let rpc_client_manager = match rpc_client_managers.get(&shard_id) {
+        // For shard-0, we'll fetch the latest metadata from the peer. Since we don't need to sync shard-0, we'll just store the metadata
```

Comment:
To confirm, this will be set to the first block we should start syncing from (which is not the latest/current block).

Reply:
right.